app/components/Article/FeedbackForm/feedbackForm.css

```diff
@@ -0,0 +1,88 @@
+.feedback-header {
+  color: var(--colors-black);
+}
+.feedback-container {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
+  box-sizing: border-box;
+  width: 384px;
+  height: 542px;
+  padding: var(--spacing-32);
+}
+.select-option {
+  border-radius: 6px;
+  background-color: #fff;
+  box-shadow: 0px 16px 40px rgba(175, 183, 194, 0.2);
+  border: 1px solid #dfe3e9;
+  box-sizing: border-box;
+  width: 320px;
+  height: 48px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 7px 17px;
+  margin: var(--spacing-16) 0;
+  cursor: pointer;
+}
+.select-option.selected {
+  border: 1px solid var(--colors-teal-500);
+}
+
+.feedback-form {
+  width: 100%;
+  position: relative;
+  height: 542px;
+  text-align: left;
+  font-size: 16px;
```

Comment:
The idea was to make it so you never have to write a line of code related to anything type. Just look at what class I use in Figma and find that same class in newroot.css

<img width="781" alt="image" src="https://github.com/StampyAI/stampy-ui/assets/825910/a8356406-f2b6-4277-9bd1-c66cbb8aa02c">